mautic/mautic
# Maxmind license key

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes, MaxMind requires license key
| Automated tests included? | Yes, they need license key in env variable in phpunit.xml.
| Related user documentation PR URL | No
| Related developer documentation PR URL | No
| Issues addressed (#s or URLs) | 
| BC breaks? | No
| Deprecations? | No

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )
The user can work with it in a standard way.

[//]: # ( Required: )
#### Description:
In back office a user can add license key

#### Steps to test this PR:
1. Try to download the file by MaxmindDownloadLookup->downloadRemoteDataStore()
2. The file should be unpacked and moved to MaxmindDownloadLookup->getLocalDataStoreFilepath()
